File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_22.java
import java.util.Scanner;

public class exercicio_22 {

    public static void main(String[] args) {
        int numero;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe um numero: ");
        numero = teclado.nextInt();

        if (numero % 2 == 0) {
            if (numero > 0) {
                System.out.printf("O numero %d eh par e positivo.\n", numero);
            } else {
                System.out.printf("O numero %d eh par e negativo.\n", numero);
            }
        } else {
            if (numero > 0) {
                System.out.printf("O numero %d eh impar e positivo.\n", numero);
            } else {
                System.out.printf("O numero %d eh impar e negativo.\n", numero);
            }
        }

        teclado.close();
    }

}

File: geek-university-algoritmo-e-logica-de-programacao/python/exercicio_22.py
numero = int(input("Informe um numero: "))

if numero % 2 == 0:
    if numero > 0:
        print("O número {0} eh par e positivo".format(numero))
    else:
        print("O número {0} eh par e negativo".format(numero))
else:
    if numero > 0:
        print("O número {0} eh impar e positivo".format(numero))
    else:
        print("O número {0} eh impar e negativo".format(numero))
